admin/class-admin.php

```diff
@@ -259,13 +259,20 @@ public function handle_actions() {
 	 */
 	protected function add_notice( $type, $message, $dismissible = true, $button = [] ) {
 		$notices            = get_option( 'wp_rest_cache_admin_notices', [] );
-		$notices[ $type ][] = [
-			'message'     => $message,
-			'dismissible' => $dismissible,
-			'button'      => $button,
-		];
-		update_option( 'wp_rest_cache_admin_notices', $notices, false );
-	}
+
+        if ( ! array_key_exists( $type, $notices ) ) {
+            $notices[ $type ] = [];
+        }
+
+        if ( ! in_array( $message, array_column( $notices[ $type ], 'message' ), true ) ) {
```

Comment:
:no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).